src/modules/85-cv/pages/monitored-service/components/Configurations/Configurations.tsx

```diff
@@ -103,6 +106,26 @@ export default function Configurations({ isTemplate, updateTemplate }: Configura
     queryParams: { accountId }
   })
 
+  const serviceRef = useRef<any | null>()
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9306981)
```suggestion
  const serviceRef = useRef<unknown | null>()
```